client/components/upgrades/gsuite/gsuite-dialog/users-form.jsx

```diff
@@ -18,7 +18,7 @@ import FormInputValidation from 'components/forms/form-input-validation';
 
 class GoogleAppsUsersForm extends React.Component {
```

Comment:
Same remark here (and the other files that were moved).

Reply:
Separate PR